webknossos-datastore/app/com/scalableminds/webknossos/datastore/explore/N5MultiscalesExplorer.scala

```diff
@@ -17,6 +17,8 @@ class N5MultiscalesExplorer(implicit val ec: ExecutionContext) extends RemoteLay
 
   override def name: String = "N5 Multiscales"
 
+  override def exploreSiblings: Boolean = false
```

Comment:
I thought we’re not doing siblings anymore?

Reply:
Also forgot to remove this here